docs/policies/policies.md

```diff
@@ -0,0 +1,515 @@
+| policy "permission" | A Policy MUST have at least one permission, prohibition, or obligation property values.
+| policy permission "action" | "use" the target Asset (under a specific permission), currently only the action "use" is used by Catena-X |
+| policy permission "constraint" | A boolean/logical expression that refines an Action and Party/Asset collection or the conditions applicable to a Rule. The leftOperand instances MUST clearly be defined to indicate the semantics of the Constraint. Catena-X will use the **left operand** of a *constraint* to associate a specific verifiable credential (VC). The left operand value will correspond to the form: [VC type].[subtype]. **Right Operand:** The rightOperand is the value of the Constraint that is to be compared to the leftOperand. All possible `Catena-X constraints` can be found here:  [https://github.com/eclipse-tractusx/ssi-docu](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/architecture/cx-3-2/edc/policy.definitions.md)
+
+### Policy Template Example with @context: `"odrl": "http://www.w3.org/ns/odrl/2/"`
+
+There is another possibility of representation: in the example above, `"@context": "http://www.w3.org/ns/odrl.jsonld"` was defined, so it is not necessary to always write `odlr:xy`. If this is not defined, the policy must look like this:
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@type": "PolicyDefinitionRequest",
+  "@id": "{{POLICY_ID}}",
+  "policy": {
+    "@type": "odrl:Set",
+    "odrl:permission": [
+      {
+        "odrl:action": {
+          "@id": "odrl:use"
+        },
+        "odrl:constraint": [
+          {
+            "@type": "odrl:LogicalConstraint",
+            "odrl:and": [
+              {
+                "odrl:leftOperand": {
+                  "@value": "<field>"
+                },
+                "odrl:operator": {
+                  "@id": "odrl:eq"
+                },
+                "odrl:rightOperand": "<value>"
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+### Policies & Verifiable Credenitals (VC)
+
+#### General Information
+
+Catena-X uses policies to determine access to and use of data. The policies refer to verifiable credentials (VC) that are stored in the Managed Identity Wallet (MIW). Catena-X uses the principle of self-sovereign identity (SSI) here.
+
+ℹ️ Detailed Information about SSI, VC and MIW can be found here:
+
+- [https://github.com/eclipse-tractusx/ssi-docu](https://github.com/eclipse-tractusx/ssi-docu)
+- [https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-2](https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-2) (for release 23.09)
+- [https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-3](https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-3) (for release 23.12)

```

Comment:
Release 23.12 will come in a few days. This page doesn't give any information. Therefore, the link should be updated or the page should be filled with information.

Reply:
Unfortunately, the page is still incomplete. But I can remove the link if that avoids confusion.

Reply:
Removed the whole section